safe_transaction_service/history/management/commands/setup_service.py

```diff
@@ -114,8 +114,8 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
         ('0x34CfAC646f301356fAa8B21e94227e3583Fe3F5F', 6399212, '1.1.1'),
     ],
     EthereumNetwork.VOLTA: [
-        # ('0x3E5c63644E683549055b9Be8653de26E0B4CD36E', 0, '1.3.0+L2'),
-        # ('0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552', 0, '1.3.0'),
+        ('0x3E5c63644E683549055b9Be8653de26E0B4CD36E', 11828662, '1.3.0+L2'),
+        ('0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552', 11942450, '1.3.0'),
```

Comment:
And for this one it seems that it was created with this tx: https://volta-explorer.energyweb.org/tx/0xa8eae8dd93a4dffdcb28d39800cad926e58a546ddfbfaf1cb35c5fc6c708f7b0/internal-transactions

So block number would be `11942451`

Reply:
Sure!